### Start new chess game.
#
# Determining who is white (and black) are accomplish by the (engine) move
# requests. White always moves first.
#

string  white_name  # white player's name 
string  black_name  # black player's name
---
int8 rc   # 0 = ok, <0 = error
